homeassistant/components/sensor/darksky.py

```diff
@@ -122,26 +131,39 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     sensors = []
     for variable in config[CONF_MONITORED_CONDITIONS]:
         sensors.append(DarkSkySensor(forecast_data, variable, name))
+        if forecast is not None and variable in ['temperature_min',
```

Comment:
Are these the only `monitored_conditions` that support forecasting? How about adding another value to the `SENSOR_TYPES` list that indicates forecast support, instead of adding a hardcoded list here?

Something like:
```
'temperature_max': ['Daily High Temperature',
                        '°C', '°F', '°C', '°C', '°C', 'mdi:thermometer', True],
```